src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -400,22 +458,32 @@ const ApplicationDetails: React.FC = () => {
 
     setIsVerifyButtonVisible(allDocsUnverified);
     setIsReverifyButtonVisible(!allDocsUnverified && anyDocFailed);
+    
+    // Check document verification status for approval/rejection
+    checkDocumentVerificationStatus(docs);
   };
 
   const setEligibilityResults = (eligibilityResponse: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)